.eslintrc.yaml

```diff
@@ -316,7 +316,6 @@ rules:
   flowtype/boolean-style: [error, boolean]
   flowtype/define-flow-type: warn
   flowtype/delimiter-dangle: off
-  flowtype/generic-spacing: [error, never]
```

Comment:
A quibble just about the commit message:

    This conflicts with our prettier config, and doesn't seem to be very
    useful. It's disabled by default upstream, and was added in a1edcaa16
    without any explanation. We frequently needed to disable it.

I think the story with ESLint plugins in general seems to be that they don't enable any rules on their own, but then they provide a "recommended" config you can extend from. This rule does seem to be in this plugin's recommended config, so that seems like the ESLint equivalent of what "enabled by default" would mean in a more familiar convention.

I think all the other reasons you give are good ones, though, and agree we should drop this rule.

(More in [the chat thread](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/fmt.2Ftest.20prettier.20mismatch/near/1327092).)

---

Looking at that commit a1edcaa16, it looks like the config it added was largely a copy of the plugin's recommended config at the time:
https://github.com/gajus/eslint-plugin-flowtype/blob/a0a6d0e49af396415731d22b55027a10265af0ac/src/configs/recommended.json

There are a few exceptions. The commit message definitely should have mentioned (a) that it was based on that recommended config, and (b) why the differences. (Or perhaps better yet, (b) would go in comments on the config itself.) The lack of any explanation is unfortunately typical of how this repo usually operated in 2017.